code/modules/antagonists/fugitive/old_god.dm

```diff
@@ -28,26 +35,26 @@
 
 /mob/camera/yalp_elor/Login()
 	..()
-	to_chat(src, "<B>My destiny was supposed to control all of humanity! My power was once absolute, but I fell out of view because of that damned Nar'Sie. At least i'm still alive...</B>")
-	to_chat(src, "<B>But nevermind that. I have only three followers left. If they perish, I will be completely forgotten and will cease to exist. I must guide the faithful.</B>")
-	to_chat(src, "<B>I sense Nanotrasen has once again tracked us, and they will reach us in about 10 minutes. I must make sure my followers are ready when they arrive.</B>")
+	to_chat(src, "<B>You must protect your followers from Nanotrasen!</B>")
+	to_chat(src, "<B>Only your followers can hear you, and you can speak to send messages to all of them, wherever they are. You can also locally whisper to anyone.</B>")
+	to_chat(src, "<B>Nanotrasen will reach you and your followers in about 10 minutes. Make sure they are ready when the time is up.</B>")
 
 /mob/camera/yalp_elor/Move(NewLoc, direct)
-	var/OldLoc = loc
-	if(NewLoc)
-		var/turf/T = get_turf(NewLoc)
-		if(locate(/obj/effect/blessing, T))
-			if((world.time - lastWarning) >= 30)
-				lastWarning = world.time
-				to_chat(src, "<span class='warning'>This turf is consecrated and can't be crossed!</span>")
-			return
-		if(istype(get_area(T), /area/chapel))
-			if((world.time - lastWarning) >= 30)
-				lastWarning = world.time
-				to_chat(src, "<span class='warning'>The Chapel is hallowed ground under a much, MUCH stronger deity, and can't be accessed!</span>")
-			return
-		forceMove(T)
-		Moved(OldLoc, direct)
+	if(!NewLoc)
+		return
+	var/turf/T = get_turf(NewLoc)
+	if(locate(/obj/effect/blessing, T))
+		if((world.time - lastWarning) >= 30)
+			lastWarning = world.time
+			to_chat(src, "<span class='warning'>This turf is consecrated and can't be crossed!</span>")
+		return
+	if(istype(get_area(T), /area/chapel))
+		if((world.time - lastWarning) >= 30)
+			lastWarning = world.time
+			to_chat(src, "<span class='warning'>The Chapel is hallowed ground under a much, MUCH stronger deity, and can't be accessed!</span>")
+		return
+	forceMove(T)
+	Moved(loc, direct)
```

Comment:
Makes no sense, you just moved so loc is not where you were.

Reply:
i was wondering why that var was nessessary....